sys/net/gnrc/network_layer/sixlowpan/iphc/gnrc_sixlowpan_iphc.c

```diff
@@ -612,20 +612,24 @@ bool gnrc_sixlowpan_iphc_encode(gnrc_pktsnip_t *pkt)
     /* check for available contexts */
     if (!ipv6_addr_is_unspecified(&(ipv6_hdr->src))) {
         src_ctx = gnrc_sixlowpan_ctx_lookup_addr(&(ipv6_hdr->src));
+        /* do not use for compression if GNRC_SIXLOWPAN_CTX_FLAGS_CID_MASK is not set */
+        if (!(src_ctx->flags_id & GNRC_SIXLOWPAN_CTX_FLAGS_COMP))
+            src_ctx = NULL;
```

Comment:
Please keep in line with the [coding conventions](https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions#indentation-and-braces).